.woodpecker/binaries.yml

```diff
@@ -2,6 +2,9 @@ depends_on:
   - test
   - web
 
+when:
```

Comment:
binaries and docker do quite the same ... and it's artifacts are only used on releases